eva-submission-docker/postgres_docker/metadata.sql

```diff
@@ -374,12 +403,12 @@ $function$
 ALTER FUNCTION evapro.eva_study_accession_trigfunc() OWNER TO metadata_user;
 GRANT ALL ON FUNCTION evapro.eva_study_accession_trigfunc() TO metadata_user;
 
-create trigger eva_study_accession after
-insert
-    or
-update
-    on
-    evapro.project for each row execute procedure evapro.eva_study_accession_trigfunc();
+-- create trigger eva_study_accession after
+-- insert
+--     or
+-- update
+--     on
+--     evapro.project for each row execute procedure evapro.eva_study_accession_trigfunc();
```

Comment:
I had to remove this trigger because it was causing the perl script to loop indefinitely.